var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -342,9 +342,19 @@ def patch(self):
 
     # The parallel compiler wrappers (i.e. h5pcc, h5pfc, etc.) reference MPI
     # compiler wrappers and do not need to be changed.
-    filter_compiler_wrappers(
-        "h5cc", "h5hlcc", "h5fc", "h5hlfc", "h5c++", "h5hlc++", relative_root="bin"
-    )
+    # These do not exist on Windows.
+    # Enable only for supported platforms.
+    for plat in ["linux", "darwin", "cray"]:
```

Comment:
plat = a plot of land.

I don't like using a variable name that has a specific meaning.

Will `OS` instead of `os` work?

Reply:
`OS` isn't really an ideal alternative for a handful of reasons. For starters it's definitely going to be distracting/confusing to see a variable with _almost_ the same name as a module, and does little to clarify things. Additionally I think simply capitalizing variable names arbitrarily to avoid naming collisions sets a bad precedent and could potentially make for messy/unreadable code. For larger reasons, it flies directly in the face of python variable naming style conventions, see https://peps.python.org/pep-0008/#function-and-variable-names.

Reply:
Typically the only things in Python code that should be all caps are constants at the module scope.

Reply:
Then, use `unix`.